test/evergreen.yml

```diff
@@ -906,7 +906,7 @@ tasks:
             set -o errexit
             set -o verbose
 
-            ${test_env_vars|} $(pwd)/test/cppsuite/run -t base_test -f test/cppsuite/configs/base_test_default.txt
+            ${test_env_vars|} $(pwd)/test/cppsuite/run -t base_test -C 'debug_mode=(cursor_copy=true)'
```

Comment:
A closed related ticket explains why I did this: WT-7526

Reply:
Its worth noting we don't want this behaviour by default as it will slow things down considerable but its useful as a test behaviour for our PR testing in evergreen to catch misuse of buffers.